src/components/Breadcrumb/Breadcrumb.less

```diff
@@ -0,0 +1,43 @@
+.lucid-Breadcrumb {
+
+	@breadcrumb-margin: 7px;
```

Comment:
can this value be derived from anything in variables.less, like `@size-XS + 1` maybe?